config/base/cert-manager/certificate.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: cert-manager.io/v1alpha2
+apiVersion: cert-manager.io/v1
```

Comment:
Do we need to update our clusters to cert-manager v1.x to keep things in line?

Reply:
Upgrading to cert-manager v1.5 (which for us is fine, as we're already past the big change at v0.14) makes this a non-issue.